Makefile

```diff
@@ -41,7 +41,7 @@ endif
 
 ifneq "" "$(shell which cargo)"
 optional-rust-support:
-	cd rust && cargo build --release
+	cd rust && RUSTFLAGS="-C target-cpu=native" cargo build --release
```

Comment:
This only affects the demangling. I guess it doesn't hurt but if the build occurs on a machine different from the run (which happens on Amazon, I don't get to choose what CPU each node type has) then I might get `SIGILL`